### DISCUSSION TITLE
Premise Types

### DISCUSSION THREAD

**Discussion by nileshtrivedi** [christianvoigt/argdown#249] [Q&A]
In most arguments, I find the claims/premises fall in a few categories:

- Objective Facts (eg: The Earth is round)
- Subjective (Personal facts) (eg: I like beer)
- Moral claims (eg: It's wrong to murder a person)

The reason why this distinction needs to be made is because of things like the Is-Ought problem. We cannot reach a moral conclusion with purely objective facts. An argument that attempts to do so would be invalid.

How would one go about encoding these types in an argument map?

**Comment by christianvoigt** [comment]
Hi @nileshtrivedi, sorry for not answering earlier, I hope you are still having fun with Argdown. You can use tags to encode different statement types in Argdown:

```argdown
<Pain & Joy>

(1) Mammals can feel pain and joy. #descriptive
(2) You should not kill an animal for fun or convenience, if it can feel pain and joy. #normative
----
(3) You should not kill mammals for fun or convenience. #normative
```

This way you have the option to [color your statements by type](https://argdown.org/guide/colorizing-maps.html) or even use [different icons](https://argdown.org/guide/adding-images.html#using-tags-to-add-images) for normative and descriptive statements.

You can hide your tags in the map by using the "removeTagsFromText" settings of the model plugin:

```
===
model:
      removeTagsFromText: true
===

<Pain & Joy>

...
```

**Comment by nileshtrivedi** [comment]
Thanks @christianvoigt!  Closing this issue as this was a question and not a bug report. You might want to enable the "Discussions" tab in repo settings to avoid getting similar questions under "Issues".

**Comment by christianvoigt** [comment]
That's a good idea. Thanks for the tip, I was not aware of Github Discussions! I will now convert all "question" issues into discussion posts.
